spec/integration_spec.rb

```diff
@@ -345,12 +347,10 @@ def rollback
 
   context "when an around hook errors early" do
     let(:organizer) {
-      build_organizer(organize: [organizer2, interactor3, organizer4, interactor5]) do
-        around do |interactor|
+      interactors = [organizer2, interactor3, organizer4, interactor5]
+      build_organizer(organize: interactors) do
+        around do |_interactor|
           raise "foo"
-          context.steps << :around_before
-          interactor.call
-          context.steps << :around_after
```

Comment:
Although it may seem obvious to Rubocop, there's some value in the assertion that these lines are _not_ called, even after a `raise`. This applies to several of the changes below as well.